blenderproc/python/types/MeshObjectUtility.py

```diff
@@ -280,8 +281,8 @@ def disable_rigidbody(self):
         if self.has_rigidbody_enabled():
             bpy.ops.rigidbody.object_remove({'object': self.blender_obj})
         else:
-            raise RuntimeError(f"MeshObject {self.get_name()} has no rigid_body component enabled")
-
+            warnings.warn(f"MeshObject {self.get_name()} has no rigid_body component enabled")
```

Comment:
Instead of removing this one could also just check before if the object has a rigid body?

```python
if obj.has_rigidbody_enabled():
    obj.disable_rigidbody()
```

Reply:
You mean in the example? I wouldn't raise an error if you disable it twice and I wouldn't want to add an if statement everytime. We do not throw an error if we enable rigidbody twice nor do we do so when hiding/unhiding an object. That's okay, no harm is being done, but the warning can be useful to find logical mistakes.

Reply:
In the function this check is already done.